tests/end-to-end/ui/14-message-popup.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-env mocha */
+
+import { adminEmail, adminPassword } from '../../data/user.js';
+import loginPage from '../../pageobjects/login.page';
+import sideNav from '../../pageobjects/side-nav.page';
+import mainContent from '../../pageobjects/main-content.page';
+
+const users = new Array(10).fill(null)
+	.map(() => `${ Date.now() }.${ Math.random().toString(36).slice(2) }`)
+	.map((uniqueId, i) => ({
+		name: `User #${ uniqueId }`,
+		username: `user.test.mentions.${ uniqueId }`,
+		email: `user.test.mentions.${ uniqueId }@rocket.chat`,
+		password: 'rocket.chat',
+		isMentionable: i % 2 === 0
+	}));
+
+const registerNewUser = user => browser.executeAsync(({ name, email, username, password, isMentionable }, done) => {
+	setTimeout(() => Meteor.call('registerUser', { name, email, pass: password }, error => {
```

Comment:
Wouldn't be quicker to use API to create users and send messages?
https://github.com/RocketChat/Rocket.Chat/blob/develop/tests/end-to-end/api/01-users.js#L31